src/redis-check-aof.c

```diff
@@ -39,6 +39,7 @@
 
 static char error[1044];
 static off_t epos;
+static long long line;
```

Comment:
```suggestion
static long long line = 0;
```
Delete the following ```line = 0;```? WDYT?

Reply:
I think it's okay and i changed it